benchmarks/browser/benchmark-append-signed.html

```diff
@@ -67,7 +75,14 @@ <h2>Results</h2>
             queriesPerSecond = 0
           }, 1000)
 
-          log = new Log(ipfs, 'benchmark-append')
+          const testKeyPath = '../test/fixtures/keys'
+          const keystore = Keystore.create(testKeyPath)
+          const identityProvider = new IdentityProvider(keystore)
+          const identitySignerFn = (key, data) => keystore.sign(key, data)
```

Comment:
I'll fix the examples and benchmarks to use the identitySigner function [as they are in the tests now](https://github.com/orbitdb/ipfs-log/pull/145/files#diff-18e937546c0d20cb7ef0cdd72434cc82R39).

Reply:
I've fixed the examples and benchmarks here https://github.com/orbitdb/ipfs-log/commit/761e9295224180e2555720d15e52afbc11095769 but can't easily PR it here, so @thiagodelgado111 take the changes if you want, but we can also add them later. Would be good @thiagodelgado111 to rebase this on master and merge to a branch on orbitdb/ipfs-log and open a new PR to track the last changes.